staging/kube-prometheus-stack/Chart.lock

```diff
@@ -10,9 +10,9 @@ dependencies:
   version: 4.21.0
 - name: grafana
   repository: https://grafana.github.io/helm-charts
-  version: 6.58.7
+  version: 6.58.10
```

Comment:
are these version bumps expected for this change?

Reply:
Im not quite sure, after executing `./upgrade_operator.sh` this file was update :thinking: